src/xcode/ENA/ENA/Source/Developer Menu/DMViewController.swift

```diff
@@ -177,6 +177,8 @@ final class DMViewController: UITableViewController, RequiresAppDependencies {
 			vc = DMDSCListsController(store: store)
 		case .crashApp:
 			vc = DMCrashAppViewController()
+		case .srs:
+			vc = DMSrsPrechecksViewController(store: store)
```

Comment:
To be consistent with the format used above

```suggestion
			vc = DMSRSPrechecksViewController(store: store)
```